### ISSUE TITLE
In microsoft powerpoint, pressing control+up and control+down do not announce that the slide was moved.

### ISSUE THREAD

**Issue by derekriemer** [nvaccess/nvda#6244]
This request is a feature request.
-------------
## STR:
1. In Microsoft Powerpoint 2016 (365), might break for older versions as well, create a new presentation.
2. Now select the new slide button, and place three slides on the page, title slide one a, slide 2 b, and slide 3 c.
3. Press shift+f6 to go to thumbnails.
4. press down arrow to put the cursor on c.
5. press control+up arrow.

### Expected: 

NVDA says "slide 3, (title) moved above slide 2 (title)

### What happens?

NVDA says nothing. the slide has been moved between a and b now.
Move a below c with the same method.

**Comment by derekriemer** [comment]
cc @michaelDCurran

**Comment by derekriemer** [comment]
Should this be labeled somehow?

**Comment by feerrenrut** [comment]
In PowerPoint 2013 I am not able to change the order of the slides with `ctrl + up`. I haven't been able to find a keyboard shortcut for re-ordering the slides in powerpoint 2013. When you move between slides however NVDA reports: 

```
Slide 2 (B)  2 of 3
Slide 3 (C)  3 of 3
```

This would be a nice feature to have, however I think it would take a fair amount of time to implement.

**Comment by Adriani90** [comment]
I agree that reporting the rearangement of slides in thumbnail pane in Powerpoint would really improve user experience when creating structured presentations.
